src/Tokenizer.js

```diff
@@ -103,7 +103,7 @@ export class Tokenizer {
       return {
         type: 'code',
         raw,
-        lang: cap[2] ? cap[2].trim() : cap[2],
+        lang: cap[2] ? cap[2].trim().replace(this.rules.inline._escapes, '$1') : cap[2],
```

Comment:
Would #2627 be achievable using something this approach, a' la DRY? #2627 works, but that regex is becoming more and more unwieldy.

Reply:
No because that has to do with finding the correct tokens. This only works for removing escapes after the token is already found.